src/builder/processor_builder.cpp

```diff
@@ -23,20 +22,50 @@
 namespace ddwaf {
 
 namespace {
+
 std::set<const scanner *> references_to_scanners(
-    const std::vector<reference_spec> &references, const indexer<const scanner> &scanners)
+    const auto &references, const indexer<const scanner> &scanners)
 {
     std::set<const scanner *> scanner_refs;
-    for (const auto &ref : references) {
+
+    // Precedence:
+    //  - Refs by ID have precedence over refs by tags
+    //  - Exclusions have precedence over inclusions
```

Comment:
Could be worded more unambiguously. Does an inclusion by id have precedence over an exclusion by tags?

(From the code I infer that the answer is yes)